src/gwt/src/org/rstudio/studio/client/workbench/views/environment/dataimport/DataImport.java

```diff
@@ -33,28 +44,58 @@
    private static DataImportUiBinder uiBinder = GWT
          .create(DataImportUiBinder.class);
    
+   private final SourceServerOperations server_;
+   private final int maxRows_ = 200;
+   
    interface DataImportUiBinder extends UiBinder<Widget, DataImport>
    {
    }
 
-   public DataImport()
+   public DataImport(SourceServerOperations server)
    {
       initWidget(uiBinder.createAndBindUi(this));
       
+      server_ = server;
+      
       Size size = DomMetrics.adjustedElementSizeToDefaultMax();
       setSize(size.width + "px", size.height + "px");
+      
+      int previewHeight = size.height - 60;
```

Comment:
@jmcphers Do you know if we can leverage one of the GWT layout management facilities to avoid having to set this height explicitly?